tsds-grafana-handler/tsds_grafana_handler.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#/usr/bin/python
```

Comment:
Why was this changed? The shebang was the correct syntax. It is not a comment :)

Reply:
Sorry. I accidentally deleted it. I fixed it.